mitiq/tests/test_folding_pyquil.py

```diff
@@ -1,48 +1,112 @@
 import numpy as np
 import random
 from pyquil import Program
-from pyquil.gates import X, Y, Z
-from mitiq.folding_pyquil import local_folding, unitary_folding, sampling_stretcher, left_stretcher
```

Comment:
More of a future point, but I think it would be good to discuss common naming conventions for (folding) functions. I generally default to function names which are verbs that (hopefully) describe what the function does, e.g. `fold_gates_from_left`, but there are different styles. Eventually we'll want consistency in folding function names, but again more of a future point.

Reply:
+1 to using verbs for function names. I like that convention.

Reply:
Agree. I will use the same names used for Cirq